### PR TITLE
DOC: update README.md with the instructions how to get submodules

### PR DESCRIPTION
**Pull request by mrakitin**
Added instructions how to get git submodules. Also, minor updates to the text.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,11 +16,18 @@ The top level makefile can be used to build the whole stack.
...
+In order to keep the release files consistent between the repositories, the `Makefile`
 target `release` is used to propagate the paths into the relevant repositories. 
 
 ## Getting started
 
+Clone the repository:
+```
+git clone https://github.com/NSLS-II/epics-modules
+cd epics-modules/
+git submodule init && git submodule update
```
Comment (mrakitin) [inline README.md]: Just realized that `make update` takes care of it. Is it worth to remove the line?

**Comment by mrakitin** [comment]
@stuwilkins, does this one deserve your attention? Thanks!

File: README.md (hunk 1 above)
Comment (stuwilkins) [reply README.md]: I think so @mrakitin the `make update` runs this and also stashes any changes, one of the unfortunate things is that the `configure/RELEASE` files are all committed to the repos, so in the act of building you make the repository dirty.

Comment (mrakitin) [reply README.md]: I removed the line `git submodule init && git submodule update` yesterday. See the updated code in the "Files changed" tab.

**Review by stuwilkins — APPROVED** [review]

**Comment by stuwilkins** [comment]
Thanks @mrakitin

**Comment by mrakitin** [comment]
Thank you for reviewing/merging!

**Pull request merged**